src/shared_modules/utils/stringHelper.h

```diff
@@ -25,6 +25,42 @@
 
 namespace Utils
 {
+    static void ISO8859ToUTF8(std::string& data)
+    {
+        // Convert from ISO-8859-1 to UTF-8
+        std::string strOut;
+        // 0xc0 is 11000000 in binary, used to check if the character is a 2-byte sequence
```

Comment:
This is the same comment than below (`... in binary, used to check if the character is a 2-byte sequence`), but this hexadecimal code is used to form the first byte in fact.